module/x/peggy/types/events.go

```diff
@@ -3,15 +3,12 @@ package types
 const (
 	EventTypeObservation               = "observation"
 	EventTypeOutgoingBatch             = "outgoing_batch"
-	EventTypeMultisigBootstrap         = "multisig_bootstrap"
 	EventTypeMultisigUpdateRequest     = "multisig_update_request"
 	EventTypeOutgoingBatchCanceled     = "outgoing_batch_canceled"
 	EventTypeOutgoingLogicCallCanceled = "outgoing_logic_call_canceled"
 	EventTypeBridgeWithdrawalReceived  = "withdrawal_received"
-	EventTypeBridgeDepositReceived     = "deposit_received"
```

Comment:
this may not be used but deposit_received is a valid event, I'll correct this during my rebase and merge

Reply:
if you could open another pull to fix the events that would be a good place to correct this and is much needed.

Reply:
Should we be keeping dead code around?

Generally I'd say no, but unsure how you approach it?